docs/how-to-guides/how-to-work.md

```diff
@@ -0,0 +1,32 @@
+# How to work on this project
+
+Before committing your changes, make sure you are working on the latest codebase by fetching or pulling to make sure you have all the work.
+
+```bash
+git checkout main
+git pull origin main
+```
+
+To initiate a build:
+
+ 1. Create a branch specific to the feature.
+
+    ```bash
+    git checkout -b <branch-name>
+    ```
+
+ 2. Make the required changes and commit
+
+    ```bash
+    git commit -m "commit-message"
+    ```
+
+ 3. Push the changes
+
+    ```bash
+    git push origin <branch-name>
+    ```
+
+For a better understanding of the entire process and standards,  please refer to Axelerant's [Git workflow.](https://engg-handbook.axelerant.com/docs/how-we-work/git/git-workflow/)
+
+N.B. If provided with a user account, you can use the management console of [platform.sh](https://platform.sh/) to handle your branch-merge requests. Please refer to the official [documentation](https://docs.platform.sh/frameworks/drupal8/developing-with-drupal.html#merge-code-changes-to-master) for further information.
```

Comment:
This is misleading. Platform.sh cannot manage merge requests. It will happen in GitHub.